src/users/your_emojis.md

```diff
@@ -0,0 +1,26 @@
+# Your Emojis
+
+There are 4 different sources from which you can use emojis through the bot:
+
+- Common Server Emojis
+- Server Cached Emojis
+- [Aliases](aliases.md)
+- [Packs](packs.md)
+
+In this section we'll explain the first two sources while you can learn more about the other two in their respective sections.
+
+## Common Server Emojis
+
+The first source of emojis are the Common Server Emojis.
+These are the emojis that have been directly uploaded onto discord servers where both you and the bot are.
+This emulates the behaviour of nitro closely as we can see in the following example:
+
+Imagine an emoji named `:nqn-nqn:` has been uploaded to server A.
```

Comment:
I'd prefer something like 'If there was an emoji named ... uploaded to server A'